teacher.rb

```diff
@@ -1,10 +1,26 @@
 require './person.rb'
 
 class Teacher < Person
-  attr_accessor :specialization
-  attr_reader :age, :name, :parent_permission
-  def initialize(specialization, age, name, parent_permission: true)
-    super(age, name, parent_permission)
+  attr_accessor :specialization, :parent_permission, :age, :name
+  
+  def initialize(specialirequire './person.rb'
+
+    class Teacher < Person
+      attr_accessor :specialization, :parent_permission, :age, :name
+      
+      def initialize(specialization, age, name, parent_permission: true)
+        super(age, name, parent_permission: parent_permission)
+        
+        @specialization = specialization
+      end
+    
+      def can_use_services?
+        true
+      end
+    end
+    zation, age, name, parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
- [ ] Kindly fix this ruby file, I am getting error, somehow the content is pasted again from lines 6 to 22.

Reply:
- [ ] Like the last code reviewer said kindly fix this file. You have the same class inside the class. I can't run the main.rb file.
![image](https://user-images.githubusercontent.com/38635158/188281753-ea894f4d-935a-4677-b3fb-55d626cfc1c6.png)